File: bubble_sort.cpp
#include <iostream>
using namespace std;


void bubblesort(int arr[], int size) {

	for (int i = 0; i < size; i++) {
		for (int j = 0; j < size - i; j++) {
			if (arr[j] > arr[j + 1])
				swap(arr[j], arr[j + 1]);
		}
	}
}
void print(int arr[], int size) {
	for (int i = 0; i < size; i++) {
		cout << arr[i] << endl;
	}
}


int main() {
	int arr[6] = { 51,23,12,76,24,52 };
	int size = 6;
	bubblesort(arr, size);
	print(arr, size);
	return 0;
}

File: BubbleSort.java
public class BubbleSort {
    public static void main(String[] args) {
        int nums[] = {7,4,10,20,12};
        for(int i=0;i<5;i++){
            for(int j=1;j<(5-i);j++){
                if(nums[j]<nums[j-1]){
                    int temp = nums[j];
                    nums[j] = nums[j-1];
                    nums[j-1] = temp;
                }
            }
        }

        for(int i=0;i<5;i++){
            System.out.print(nums[i] + " ");

        }
    }
}
